### PR TITLE
Bugfixes

### PR DESCRIPTION
**Pull request by Hammarn**
Added back the `=` in `ClusterOptions` so that the config file works.
Changed `errorStrategy warning `(that doesn't exist)  to the new `finish`

### CODE REVIEW

File: main.nf [hunk 1]
```diff
@@ -131,7 +131,7 @@ process fastqc {
      memory { 2.GB * task.attempt }
      time { 4.h * task.attempt }
      
-     errorStrategy { task.exitStatus == 143 ? 'retry' : 'warning' }
+     errorStrategy { task.exitStatus == 143 ? 'retry' : 'terminate' }
```
Comment (ewels) [inline main.nf]: Why `terminate` here? Better to have `ignore`?

File: main.nf [hunk 2]
```diff
@@ -527,7 +527,7 @@ process sample_correlation {
      
      memory { 16.GB * task.attempt }
      time { 2.h * task.attempt }
-     errorStrategy { task.exitStatus == 143 ? 'retry' : 'warning' }
+     errorStrategy { task.exitStatus == 143 ? 'retry' : 'finish' }
```
Comment (ewels) [inline main.nf]: Do you know what happens to these processes if one or more steps has failed previously? I guess we don't want them to run if anything is missing.

File: main.nf (hunk 1 above)
Comment (Hammarn) [inline main.nf]: Agreed!

File: main.nf (hunk 2 above)
Comment (Hammarn) [inline main.nf]: Good point, change those to terminate?

Comment (ewels) [inline main.nf]: Actually, I think it should be fine. The docs read:
> Initiates an orderly pipeline shutdown when an error condition is raised, waiting the completion of any submitted job.

So I don't think any _new_ processes should be kicked off.

**Pull request merged**